ansible/roles/ansible-tvault-contego-extension/templates/s3_others.conf.j2

```diff
@@ -0,0 +1,24 @@
+[DEFAULT]
+vault_storage_type = s3
+vault_storage_nfs_export = TrilioVault
+vault_data_directory_old = {{TVAULT_DATA_DIR_OLD}}
+vault_data_directory = {{TVAULT_DATA_DIR}}
+log_file = /var/log/nova/tvault-contego.log
+debug = False
+verbose = True
+max_uploads_pending = 3
+max_commit_pending = 3
+vault_s3_auth_version = DEFAULT
+vault_s3_access_key_id = {{VAULT_S3_ACCESS_KEY}}
+vault_s3_secret_access_key = {{VAULT_S3_SECRET_ACCESS_KEY}}
+vault_s3_region_name = us-east-1
```

Comment:
Need to have a variable for region name as well.

Reply:
Requested changes done